extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -766,12 +766,24 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     // Force size of frame to match creative or, if creative size is unknown,
     // the slot. This ensures that the creative is centered in the former case,
     // and not truncated in the latter.
-    // TODO(levitzky) Figure out the behavior of responsive + multi-size.
     const size = this.returnedSize_ || this.getSlotSize();
+    const isMultiSizeFluid = this.isFluid_ && this.returnedSize_ &&
+        // TODO(@glevitzky, 11583) Remove this clause once we stop sending back
+        // the size header for fluid ads. Fluid size headers always come back as
+        // 0x0.
+        !(size.width == 0 && size.height == 0);
```

Comment:
Out of curiosity, what happens in between when we stop sending that back, and when we remove this line? It will still serve, but be left-justified?

Reply:
The goal is to get multi-size ad slots to be left-justified and fluid ad slots centered. Once this change goes in, that will be the case, and that should still be the case in the interim between when we stop sending the header back and when we remove this line.